### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### In these milestone:
**I added the following:**

- Html
- Css
- Linters

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
+
```
Comment (DanteAlonsoHT) [inline .gitignore]: I strongly recommend you to include `test.md` in your `.gitignore` in order to have better practices for future projects. 😉 

- [ ]  Add `test.md` inside your `.gitignore` file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
+every time.
+
+
+### AUTHOR
+
+`Gabriel Nwachukwu`
```
Comment (DanteAlonsoHT) [inline README.md]: You should use the README template provided by Microverse, it's good if you want to adjust some of the most important instructions inside the README template format.

- [ ]  Use the [README template](https://github.com/microverseinc/readme-template) provided by Microverse

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

**Hi @gabrielcoder247,** 

You did a job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

- [ ] Create a `css` folder where you can put any .css file in order to have better practices in your repositories.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- Always having a good and descriptive README file for your repositories.
- Use the Grammarly app to avoid issues with orthography (Pull request description).

<hr>

I know you have the capabilities enough to make these changes required in time 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @gabrielcoder247, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Say Hello Microverse
```
Comment (mke2111) [inline README.md:3]: > You have done great to use the recommended readme template. 👍🏽 

- [ ] Now let use personalize this readme to your full description and ownership. 
- Add a short description message about the project features you implemented.
- Add clear instructions on how to get started including the necessary commands for the terminal.
- Under the getting started section, remove some sub-titles you have not used or implemented such as the `tests`, `deployment`, and others.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/gabrielcoder247)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gabriel-nwachukwu-209613173/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (mke2111) [inline README.md]: - Add the appropriate issues link for your repo in this file.

**Review by carlos-ssh — APPROVED** [review]
Hi @gabrielcoder247 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**